mongabay/supply-chain-chart
# Project cleanup

This PR is a general cleanup of the tool:
- Remove code from the satellite stamp and unused libraries
- Remove the accordion from the sidebar and fix its width
- Add the attributions to Trase and make the visualisation responsive
- Fix the styles of the success modal
- Update the description of the tool and the README

## Testing instructions

1. Open the tool
2. Play with the filters

The visualisation should update as before and the sidebar's width should be fixed.

3. Resize the window

The visualisation should maintain the same aspect ratio.

## Pivotal Tracker

Not tracked.
